packages/mui-material/scripts/rollup.config.mjs

```diff
@@ -117,6 +117,13 @@ const nestedFolder = {
       return resolveNestedImport('mui-system', importee);
     }
 
+    if (importee.indexOf('react/jsx-runtime') === 0) {
+      return path.resolve(
```

Comment:
would this work with `createRequire(import.meta.url).resolve('react/jsx-runtime')`?

Reply:
@Janpot `createRequire(import.meta.url).resolve('react18/jsx-runtime')` works 👍🏼, updated